tokio-postgres/src/connect_raw.rs

```diff
@@ -141,8 +141,13 @@ where
     T: AsyncRead + AsyncWrite + Unpin,
 {
     match stream.try_next().await.map_err(Error::io)? {
-        Some(Message::AuthenticationOk) => return Ok(()),
+        Some(Message::AuthenticationOk) => {
+            no_channel_binding(config)?;
```

Comment:
I feel like the name here is confusing and it's a slightly different case than password, md5, etc. Really what you want to know here is "is the client satisfied with the authenticity of the server?". Maybe something like ``check_server_auth()`` or something?